app/src/main/java/com/sopt/now/LoginActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.sopt.now
+
+import android.content.Intent
+import android.os.Bundle
+import android.service.autofill.UserData
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.snackbar.Snackbar
+import com.sopt.now.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        fun showSnacker(view: View, message: String) {
+            Snackbar.make(
+                view,
+                message,
+                Snackbar.LENGTH_SHORT
+            ).show()
+        }
+        setContentView(binding.root)
+        val userData = intent.getParcelableExtra<UserDataInput>(INTENT_USER_DATA)
+
+        val signId = userData?.getUserSignUpId()
+        val signPw = userData?.getUserSignUpPw()
```

Comment:
```suggestion
        val signId = userData?.userId
        val signPw = userData?.userPW
```
요친구들만 이렇게 사용하면 될것 같아요!
data class에 선언된 변수들을 그대로 가져와서 쓰시면 됩니다.
굳이 return함수 만들 필요는 없어요!

data class의 내부 함수에 값을 return하는 형태는 아마 추후에 쓰일테니 알아둬 두시기만 하면 됩니다!